app/views/works/_uploads_form.html.erb

```diff
@@ -33,7 +33,7 @@
                     </td>
                     <td><span><%= upload.created_at %></span></td>
                     <td>
-                      <input id="work-deposit-uploads" name="work[replaced_uploads][<%= upload.id %>]" type="file" />
+                      <input id="work-deposit-uploads-<%= upload.id %>" name="work[replaced_uploads][<%= upload.key %>]" type="file" />
```

Comment:
Just looking at this, I don't have a clean sense of what `id` is for, and what `key` is for.

Reply:
@mccalluc I'm matching delete for consistency. Key in ActiveStorage is the file path in s3 (or other storage back ends) and id is the id of the database record pointing to S3